dotCMS/src/main/java/com/dotcms/rest/api/v1/site/SiteHelper.java

```diff
@@ -177,4 +177,18 @@ public Host getCurrentSite(final HttpServletRequest req, final User user) {
 			throw new DotRuntimeException(e);
 		}
 	}
+
+	public void swicthSite(HttpServletRequest req, String hostId) {
+		final HttpSession session = req.getSession();
+
+		session.setAttribute(
+				com.dotmarketing.util.WebKeys.CMS_SELECTED_HOST_ID, hostId);
+		session.removeAttribute(WebKeys.CONTENTLET_LAST_SEARCH);
+	}
+
+	public Host swicthToDefaultHost(HttpServletRequest req, User user) throws DotSecurityException, DotDataException {
```

Comment:
This method is misspelled: "swicthToDefaultHost", should be "switchToDefaultHost"